BackEndFlask/models/schemas.py

```diff
@@ -95,6 +95,7 @@ class UserCourse(db.Model):
     user_course_id = db.Column(db.Integer, primary_key=True)
     user_id = db.Column(db.Integer, ForeignKey(User.user_id), nullable=False)
     course_id = db.Column(db.Integer, ForeignKey(Course.course_id), nullable=False)
+    active = db.Column(db.Boolean, nullable=True) 
```

Comment:
Why allow active to be nullable? What does it mean if a user has a null active? Seems like we should be able to say all users are either active or not.